x-pack/plugins/ml/public/explorer/explorer.html

```diff
@@ -115,22 +154,34 @@ <h4 class="euiTitle euiTitle--small">No {{swimlaneViewByFieldName}} influencers
           <br /><br />
         </div>
   
-        <span class="panel-title euiText">
-          Anomalies
-        </span>
+        <span
+          class="panel-title euiText"
+          18n-id="xpack.ml.explorer.anomaliesTitle"
+          i18n-default-message="Anomalies"
+        ></span>
 
         <div class="euiFlexGroup euiFlexGroup--gutterLarge euiFlexGroup--directionRow euiFlexGroup--responsive ml-anomalies-controls">
           <div class="euiFlexItem euiFlexItem--flexGrowZero" style="width:170px">
             <div class="euiFormRow" id="select_severity_control">
-              <label class="euiFormLabel" for="select_severity">Severity threshold</label>
+              <label
+                class="euiFormLabel"
+                for="select_severity"
+                18n-id="xpack.ml.explorer.severityThresholdLabel"
+                i18n-default-message="Severity threshold"
+              ></label>
```

Comment:
This doesn't get rendered correctly, the string "Severity threshold" doesn't end up as inner HTML/text of the label element.

![image](https://user-images.githubusercontent.com/230104/50635089-16280580-0f51-11e9-9f02-3f2254a38c64.png)

![image](https://user-images.githubusercontent.com/230104/50635113-263fe500-0f51-11e9-8754-48b353f0ec1b.png)